modules/react-components/src/components/renderer/markdown/components/image.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Box from "@oxygen-ui/react/Box";
+import { MarkdownCustomComponentPropsInterface } from "@wso2is/react-components";
+import classNames from "classnames";
+import React, { FunctionComponent, ReactElement } from "react";
+import "./image.scss";
+
+/**
+ * Props interface for the `MarkdownImage` component.
+ */
+interface MarkdownImageProps extends MarkdownCustomComponentPropsInterface<"img"> {
+    /**
+     * Custom attributes supplied by the 'rehype-attr' plugin.
+     */
+    "data-config"?: {
```

Comment:
add support to hide the boarder

Reply:
https://github.com/wso2/identity-apps/pull/5896/commits/4630290477fe28a513b8bcef886969d74c6d6156